pkg/userdata/rhel/provider_test.go

```diff
@@ -0,0 +1,258 @@
+/*
+Copyright 2019 The Machine Controller Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+//
+// UserData plugin for RHEL.
+//
+
+package rhel
+
+import (
+	"flag"
+	"net"
+	"testing"
+
+	clusterv1alpha1 "github.com/kubermatic/machine-controller/pkg/apis/cluster/v1alpha1"
+	"github.com/kubermatic/machine-controller/pkg/apis/plugin"
+	testhelper "github.com/kubermatic/machine-controller/pkg/test"
+	"github.com/kubermatic/machine-controller/pkg/userdata/convert"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+)
+
+var (
+	update = flag.Bool("update", false, "update testdata files")
+
+	pemCertificate = `-----BEGIN CERTIFICATE-----
```

Comment:
Is it obvious what this certificate is or should we leave a comment?

Reply:
This is a certificate which actually used in all our userdata providers testing